rascal2/ui/presenter.py

```diff
@@ -148,6 +149,15 @@ def ask_to_save_project(self):
 
         return proceed
 
+    def export_results(self):
+        """Export the results object."""
+        if self.model.results:
+            save_file = self.view.get_save_file("Export Results", "", "*.json")
```

Comment:
I think we should provide a default file name i.e.
`save_file = self.view.get_save_file("Export Results", "result", "*.json")`

Reply:
The code you've quoted there would give a default directory of "result" wouldn't it? We probably shouldn't use "result" as a default filename since `results.json` is the file used to save the results as standard.

Reply:
Yeah, we can use a different name